hpx/plugins/binary_filter_factory_base.hpp

```diff
@@ -10,8 +10,8 @@
 #include <hpx/plugins/plugin_factory_base.hpp>
 #include <hpx/runtime/serialization/binary_filter.hpp>
 
-#include <hpx/util/plugin.hpp>
-#include <hpx/util/plugin/export_plugin.hpp>
+#include <hpx/plugin.hpp>
```

Comment:
Should we try to do the same as with the other modules where `hpx/modulename.hpp` just includes all (public) headers from the module (`GLOBAL_HEADER_GEN ON`)? And maybe rename this one `hpx/plugin/plugin.hpp`?

Reply:
Sorry, no reason. I thought the current `hpx/plugin.hpp` contained something in addition to just including other headers, but I see that's not the case.